std_msgs/Header header

float32 vx		# x velocity [m/s]
float32 vy		# y velocity [m/s]
float32 vz		# z velocity [m/s]
float32 yaw_rate 	# yaw rate [deg/s]
